xbmc/input/joysticks/DeadzoneFilter.h

```diff
@@ -45,7 +45,7 @@ namespace JOYSTICK
    *
    *   - Negative in the interval [-1.0, -deadzone)
    *   - Zero in the interval [-deadzone, deadzone]
-   *   - Positive in the interval (deadzone, 1.0)
+   *   - Positive in the interval (deadzone, 1.0]
```

Comment:
Nope, it's correct because it's a mathematical term specifying whether the value itself (in this case `1.0`) is inclusive or exlusive.